config/skyAnalysis.py

```diff
@@ -2,5 +2,4 @@
 
 
 config.colorterms.load(os.path.join(os.path.dirname(__file__), "colorterms.py"))
-config.refObjLoader.load(os.path.join(os.path.dirname(__file__), "..", "filterMap.py"))
```

Comment:
Oh, actually, as of https://github.com/lsst-dm/pipe_analysis/commit/ebf9570b99a1d2bf3845d8ef11084c4d3a3a72a5, this can be deleted (sorry, I should've done it on that ticket, so feel free to punt!).

Reply:
This line now reads:
`config.refObjLoader.load(os.path.join(os.path.dirname(__file__), "filterMap.py"))`
i.e., with the ".." removed. Am I still okay to remove here following your comment?

Reply:
Yeah, the task itself is now gone, so this is a config file to nowhere...

Reply:
Removed. This file now only loads `colorterms.py`.